control_panel_api/views.py

```diff
@@ -157,10 +155,8 @@ def post(self, request, *args, **kwargs):
         serializer = self.get_serializer(data=request.data)
         serializer.is_valid(raise_exception=True)
 
-        Auth0().add_group_member(
-            self.get_object().slug,
-            serializer.validated_data['email']
-        )
+        app = self.get_object()
+        app.add_customers([serializer.validated_data['email']])
```

Comment:
How does this `validated_data['email']` works when posting a JSON array like in this case? I'm not quite following what's happening here sorry (how the POSTed data is returned, how you convert to a list of emails, why there are `[]` around this expression, etc...)

Reply:
`validated_data['email']` is a `str`, not a JSON array.
I've written `App.add_customers` to take a list of email addresses as the next thing I will do is implement bulk adding customers, but this view still only permits adding one customer at a time.